test/recipes/30-test_evp_data/evpkdf_ss.txt

```diff
@@ -280,36 +280,42 @@ Ctrl.hexsecret = hexsecret:ebe28edbae5a410b87a479243db3f690
 Ctrl.hexinfo = hexinfo:e60dd8b28228ce5b9be74d3b
 Output = b4a23963e07f485382cb358a493daec1759ac7043dbeac37152c6ddf105031f0f239f270b7f30616166f10e5d2b4cb11ba8bf4ba3f2276885abfbc3e811a568d480d9192
 
+Availablein = default
```

Comment:
Maybe use FIPSversion = <3.4.0 instead?

Reply:
If we use `FIPSversion = <3.4.0`, it means that the test is not checked for default provider.

If someone doesn't build FIPS provider, the test will be passed directly. Therefore, I use `Availablein = default` here.

Reply:
Not checking against the default provider if the FIPSversion is specified is only temporary.  The check will be put back once all the indicator churn is over.  These can be fixed as part of that process IMO.